src/App.js

```diff
@@ -83,17 +95,36 @@ const App = () => {
           <Route path="/login" element={<LoginPage />} />
           <Route path="/Register" element={<Register />} />
           <Route path="/ImprintPage" element={<ImprintPage />} />
+          <Route
```

Comment:
Move these files too inside the if condition below

Reply:
Done